Framework/Kernel/inc/MantidKernel/ErrorReporter.h

```diff
@@ -60,7 +59,7 @@ class MANTID_KERNEL_DLL ErrorReporter {
   /// Target url
   std::string m_url;
   /// recovery file path
```

Comment:
Can we either update the comments or remove them for all of these member variables? The names are good so I think the comments are not really required.